src/Controls/src/Build.Tasks/XamlCache.cs

```diff
@@ -74,6 +74,8 @@ public ICollection<string> GetResourceNamesInUse(VariableDefinition variableDefi
 		{ module.ImportReference(this, ("Microsoft.Maui", "Microsoft.Maui.Converters", "FlexAlignSelfTypeConverter")), typeof(EnumTypeConverter<Layouts.FlexAlignSelf>) },
 		{ module.ImportReference(this, ("Microsoft.Maui", "Microsoft.Maui.Converters", "FlexWrapTypeConverter")), typeof(EnumTypeConverter<Layouts.FlexWrap>) },
 		{ module.ImportReference(this, ("Microsoft.Maui", "Microsoft.Maui.Converters", "FlexBasisTypeConverter")), typeof(FlexBasisTypeConverter) },
+		// { module.ImportReference(this, ("Microsoft.Maui", "Microsoft.Maui", "GridLength.GridLengthTypeConverter")), typeof(GridLengthTypeConverter) },
```

Comment:
this should be fixed (ImportReference should accept nested types) but of no importance as GridLengthTypeConverters are almost never use directly, but through collections